imports/components/@primitives/UI/live/index.js

```diff
@@ -79,14 +81,24 @@ class LiveWithoutData extends Component {
     const { live, show, embedCode } = this.props.live;
     if (!live || !show || !embedCode) return null;
 
+    // TODO load different embed code for beta users
+
+    // create beta link
+    const shouldShowBetaLink = (
+      this.props.person
+      && !this.props.person.authLoading
+      && this.props.person.authorized
+    );
+    const link = shouldShowBetaLink ? `/wowza/${embedCode}` : `/video/${embedCode}`;
+
     return (
       <Motion
```

Comment:
@JakeDawkins move the logic of find a link to a method on the class and use it inside the render function. Then you can test the method